tests/testthat/test-utilities-simulation.R

```diff
@@ -1,378 +1,378 @@
-context("exportIndividualSimulations")

```

Comment:
@IndrajeetPatil This is happening again. I think RStudio is doing some weird stuff. Hopefully this won't happen again with this change. Merging since this is the same as in the other repo

Reply:
Yeah, I noticed 😅 

But ignoring the whitespace change is quite the neat trick to avoid any headache that might arise from this while reviewing the PR, so I am not too concerned about this anymore.

I wish this was GitHub's default settings for showing code changes.